meta/Decays.m

```diff
@@ -766,7 +766,7 @@ would violate Ward identity (as at the level of dim 4
                       If[numIndices == 0 || dim <= 1,
                          result = result <> " {};\n";,
                          result = result <> " {{" <> ToString[indexVal] <>
-                                  StringJoin[Table[", 0", {i, 1, numIndices - 1}]] <> "}};\n";
+                                  StringJoin[ConstantArray[", 0", numIndices - 1]] <> "}};\n";
```

Comment:
Didn't know, that this function exists. I think, that now I need to change some code as well :)